src/py42/services/_connection.py

```diff
@@ -191,16 +191,31 @@ def request(
                 proxies=proxies or settings.proxies,
             )
 
-            if not stream and response is not None:
-                # setting this manually speeds up read times
-                response.encoding = u"utf-8"
-
-            if response is not None and 200 <= response.status_code <= 399:
-                return Py42Response(response)
-
-            if response is not None and response.status_code == 401:
-                if isinstance(self._auth, C42RenewableAuth):
-                    self._auth.clear_credentials()
+            if response is not None:
+                if not stream:
+                    # setting this manually speeds up read times
+                    response.encoding = u"utf-8"
+
+                    debug.logger.error(
```

Comment:
These should be logged at debug level, not error. We don't want _every_ response sent to stderr by default, only when the user has enabled debug logging.

Reply:
based on this [conversation](https://github.com/code42/py42/pull/320#issuecomment-825029146) I set it to error.